Tests/conf.json

```diff
@@ -20,6 +20,20 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "integrations": [
+                "WildFire-v2",
+                "Cortex XDR - IR",
+                "Joe Security"
+            ],
+            "instance_names": [
+                "WildFire-v2",
+                "Cortex XDR - IR",
+                "Joe_Security"
```

Comment:
We need to set instance names only if we have multiple credentials for the same integration, this doesn't seem the case so we can remove this.

Reply:
Fixed, thanks.